corehq/apps/userreports/reports/filters.py

```diff
@@ -52,7 +52,9 @@ def __init__(self, filter, value):
     def to_sql_filter(self):
         if self.value is None:
             return ""
-        return "{0} {1} :operand".format(self.filter.field, self.value['operator'])
+        ret = BasicFilter(self.filter.field, 'operand')
+        ret.operator = self.value['operator']
```

Comment:
perhaps it would be better to add operator to this: https://github.com/dimagi/sql-agg/blob/master/sqlagg/filters.py#L20

Wasn't sure on our process behind updating things we use via pip